deploy/deploy.sh

```diff
@@ -16,54 +16,122 @@
 set -e
 set -o pipefail
 
+# when not empty it will run CSI driver on read-only mode
+READ_ONLY_MODE="${1}"
```

Comment:
Will probably want to update this based on what we call the flag.

Reply:
+1

Reply:
Please consider latest changes.